pytest_ibutsu.py

```diff
@@ -256,6 +260,7 @@ def add_run(self, run=None):
             run["id"] = str(uuid.uuid4())
         if not run.get("source"):
             run["source"] = self.source
+        self.set_run_id(run["id"])
```

Comment:
Should this maybe be moved to somewhere the plugin is re-initialised with by xdist?

Reply:
I'm not sure about this one, when I moved the `temp_path` to a property, it was failing because the `run_id` wasn't set yet, moving it here seemed to allow it to work.

Reply:
OK I was able to remove this by using `get_temp_path`, which the first go round will use `run["id"]`